pkg/quay/quay_debug_test.go

```diff
@@ -212,24 +214,37 @@ func TestCreateNotification(t *testing.T) {
 	if quayNotification.UUID == "" {
 		t.Fatal("Notification UUID should not be empty")
 	}
+}
 
-	allNotifications, err := quayClient.GetNotifications(quayOrgName, quayImageRepoName)
```

Comment:
trying to GetNotifications should be after each method

create/update expecting some to exist
delete  expecting non to exist

Reply:
That was originally present in the code but removed due to a code review. I'll add it back